### DISCUSSION TITLE
Global Registration open3D

### DISCUSSION THREAD

**Discussion by robertoruizmu** [isl-org/Open3D#3480] [Q&A] [answered]
I´m trying this global registration tutorial [http://www.open3d.org/docs/release/tutorial/pipelines/global_registration.html?highlight=ransac#fast-global-registration](url) , but I don´t know what values should I use for parameters like distance threshold in ICP registration and RANSAC, also in voxel size or CorrespondenceCheckerBasedOnEdgeLength. I mean, what kind of measure is it ? mm, cm, m? do they depend on the point cloud? What is a high or low value ? How can I set the correct value? I used the values in the tutorial, and changed them, but I can´t get a good aligment.
Thanks everyone !

**Comment by theNded** [comment, marked as answer]
`voxel_size` is in meter. The recommended value is 0.05 for a coarse scale and 0.0125 for a fine scale.
`CorrespondenceCheckerBasedOnEdgeLength` is a relative measurement, so metric doesn't matter.
RANSAC is really sensitive to data, as it is based on feature correspondences. Unfortunately there is no recommended setup that rules all data as of today.

**Reply by robertoruizmu** [reply]
thank you !

**Reply by germanros1987** [reply]
Please, @robertoruizmu consider marking this discussion as answered.
